src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -92,6 +92,24 @@ DiagnosticBag diagnostics
             return new TupleTypeSymbol(underlyingType, elementNames, binder.Compilation.Assembly);
         }
 
+        internal static TupleTypeSymbol Create(
+            ImmutableArray<TypeSymbol> elementTypes,
+            ImmutableArray<string> elementNames,
+            CSharpCompilation compilation)
+        {
+            Debug.Assert(elementNames.IsDefault || elementTypes.Length == elementNames.Length);
+
+            int numElements = elementTypes.Length;
+
+            if (numElements <= 1)
+            {
+                throw ExceptionUtilities.Unreachable;
```

Comment:
Why is this unreachable?  It seems very possible that someone coudl pass in <= 1 elements.  If that's not allowed, we should throw a real argument exception.